runtime/buildtools.mk

```diff
@@ -49,8 +49,8 @@ else
 	JAVA := $(if $(J9_ROOT),java8,$(DEV_TOOLS)\ibm-jdk-1.8.0\bin\java)
 endif
 
-ifdef VERSION_MAJOR
-	EXTRA_CMAKE_ARGS += -DJ9VM_JAVA_VERSION=$(VERSION_MAJOR)
+ifneq (,$(VERSION_MAJOR))
+	EXTRA_CMAKE_ARGS += -DJAVA_SPEC_VERSION=$(VERSION_MAJOR)
```

Comment:
It doesn't seem right to stop defining J9VM_JAVA_VERSION for cmake, at least without fixing all the references. Its used in platform.cmake and version.cmake

Reply:
It seems my search for for `J9VM_JAVA_VERSION` was incomplete: I'll fix those references.